src/playbooks/deploy/deploy.yaml

```diff
@@ -66,5 +62,4 @@
     - candlepin
     - httpd
     - pulp
-    - foreman_proxy
```

Comment:
should the role (and its tests) be completely deleted?

Reply:
Yes, we can find it again in history if we need.

Reply:
done